src/count.js

```diff
@@ -20,6 +20,9 @@ function count(ast) {
     case 'keyframes':
     case 'import':
     case 'supports':
+    case 'charset':
+    case 'namespace':
+    case 'page':
```

Comment:
Finally got what you mean (how the switch, return 0 works). So I've tested it out below and the `@page` should be fine, the other rules (`charset`, `namespace`) should be fine since its conditions are like that of `import`.

reference: [CSS Compatibility in Internet Explorer](https://msdn.microsoft.com/en-us/library/hh781508(v=vs.85).aspx)

===

switch type | notes (IE conditions)
------------ | -------------
`comment` |
`font-face` |
`keyframes` |
`import` | should occur at the start of a style sheet, before any declarations
`supports` |
`charset` | **(new)** must be at the top of the file, 1st line (same case as import)
`namespace` | **(new)** must declare an @namespace rule after any @charset or @import rules (same case as import)
`page` | **(new)** IE 8+, can be declared anywhere in the file

#### OBJECTIVES
 - test if `@page` is not affected by the 4095 selector limit

==

### TEST 1
- single `@page` declaration

```css
#test,
...,
#test { color: #900; }
@page :first {
  margin: 2in 3in;
}
```

##### @page after 4095th Selector
OS / Ver  | Expected | Actual | .
--------- | -------- | ------ | ----
XP / IE 6 | Fail     | Fail   | :o:
XP / IE 7 | Fail     | Fail   | :o:
XP / IE 8 | Pass     | Pass   | :o:
W7 / IE 8 | Pass     | Pass   | :o:
W7 / IE 9 | Pass     | Pass   | :o:

##### @page as 1st in line
OS / Ver  | Expected | Actual | .
--------- | -------- | ------ | ----
XP / IE 6 | Fail     | Fail   | :o:
XP / IE 7 | Fail     | Fail   | :o:
XP / IE 8 | Pass     | Pass   | :o:
W7 / IE 8 | Pass     | Pass   | :o:
W7 / IE 9 | Pass     | Pass   | :o:

==

### TEST 2
- multiple `@page` declaration

```css
#test,
...,
#test { color: #900; }
@page :left {
    margin-left: 3in;
    margin-right: 1in;
}

@page :right {
    margin-left: 1in;
    margin-right: 3in;
}
```

##### @page after 4095th Selector
OS / Ver  | Expected | Actual | .
--------- | -------- | ------ | ----
XP / IE 6 | Fail     | Fail   | :o:
XP / IE 7 | Fail     | Fail   | :o:
XP / IE 8 | Pass     | Pass   | :o:
W7 / IE 8 | Pass     | Pass   | :o:
W7 / IE 9 | Pass     | Pass   | :o:

##### @page as 1st in line
OS / Ver  | Expected | Actual | .
--------- | -------- | ------ | ----
XP / IE 6 | Fail     | Fail   | :o:
XP / IE 7 | Fail     | Fail   | :o:
XP / IE 8 | Pass     | Pass   | :o:
W7 / IE 8 | Pass     | Pass   | :o:
W7 / IE 9 | Pass     | Pass   | :o:

==

### TEST 3 (Test to Fail)
- 4096 selectors
- multiple `@page` declaration (same as Test 2 code)

##### @page after 4096th Selector
OS / Ver  | Expected | Actual | .
--------- | -------- | ------ | ----
XP / IE 8 | Fail     | Fail   | :o:
W7 / IE 8 | Fail     | Fail   | :o:
W7 / IE 9 | Fail     | Pass   | :x:

##### @page after 4100th single declaration/rule

```css
.redClass0001 { background: red }
.redClass#### { background: red }
@page :left {
    margin-left: 3in;
    margin-right: 1in;
}

@page :right {
    margin-left: 1in;
    margin-right: 3in;
}
```

OS / Ver  | Expected | Actual | .
--------- | -------- | ------ | ----
W7 / IE 9 | Fail     | Fail   | :o:

==

### Conclusion
`@page` will still work if declared after the 4095th selector, but will not work if declared after 4096th selector, as expected, except for the case in Test 3 (IE9).

==

Hope this helps!